### PR TITLE
Add msisdns endpoint

### PR DESCRIPTION
**Pull request by Timvde**
This endpoint can be used to retrieve whether the given msisdn is in our
customer base.

### CODE REVIEW

File: swagger.yaml [hunk 1]
```diff
@@ -27,6 +27,8 @@ tags:
     description: Finance
   - name: Mandates
     description: Mandates
+  - name: Msisdns
```
Comment (pieterdd) [inline swagger.yaml]: Nitpick: I think the correct spelling is MSISDNs.

Comment (Timvde) [reply swagger.yaml]: It is, but we never wrote it like that. For consistency sake, I prefer to keep it like this. If you want to rewrite it, you should open a request to do it everywhere.

File: paths/index.yaml [hunk 2]
```diff
@@ -103,3 +103,5 @@
   $ref: ./loyalty/loyalty_transfer.yaml
 /loyalty-points/transactions:
   $ref: ./loyalty/loyalty_transactions.yaml
+/msisdns:
```
Comment (pieterdd) [inline paths/index.yaml]: Is this endpoint URL final? Other options include `/is-registered-msisdn`, `/msisdn-check` or `/check-msisdn`.

Comment (Timvde) [reply paths/index.yaml]: Renamed. Check again?

Comment (pieterdd) [reply paths/index.yaml]: Checked again.

**Comment by Timvde** [comment]
Latest force push only changes the commit message to update the name of the endpoint.

**Pull request merged**